src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,14 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+import './TodoInfo.scss';
+
+export const TodoInfo = ({ todo: { completed, title, user } }) => (
+  <article className={cn('TodoInfo', {
+    'TodoInfo--completed': completed === true,
+  })}
+  >
+    <h2 className="TodoInfo__title">{title}</h2>
+
+    <UserInfo user={user} />
```

Comment:
the `user` might be `null` (see `getUserById`) - handle this case to avoid possible errors

Reply:
Correct. Thank you.